spec/models/concerns/wakeable_spec.rb

```diff
@@ -118,6 +118,21 @@ def my_method
           expect(two).to have_wakes_graph(:canonical_location => '/two/some-new-title',
                                           :legacy_locations => ['/two/some-title'])
         end
+
+        it 'initializes a new wakes graph if for some reason none exists' do
+          wakeable = model_class.create(:title => 'Some Title')
```

Comment:
i agree. ideally, we'd do it in between work on wakes, when there are no open branches